### DISCUSSION TITLE
Solving coding error

### DISCUSSION THREAD

**Discussion by oye-bobs** [cu-esiil-edu/2024-data-short-course#50] [Q&A]
Hi Everyone, I just ran into a problem when trying to solve the jupyter notebook exercises. specifically when trying to clean the files in the data frame and call it. I was able to clean it to display those columns present in the screen shot
![Screenshot 2024-04-18 122831](https://github.com/cu-esiil-edu/2024-data-short-course/assets/33554816/13eb4479-2a82-46f9-9fc5-a11c8b9a3e35).

 but while trying to test the cell, I got an error
![lll](https://github.com/cu-esiil-edu/2024-data-short-course/assets/33554816/d9522565-5a85-49a6-984f-4600295e70c2)

I would appreciate if someone could walk me through it.

**Comment by eculler** [comment]
@oye-bobs Without seeing the whole error it is hard to tell. But I think this is a problem with the test cell, and not with your code. Can you send the entire error message so I can try to reproduce your error and fix it in a future version?

**Reply by oye-bobs** [reply]
@eculler Thanks for your response, I found a way to solve it.

Apparently in creating the Boulder DF, I included some cells that should'nt have been included, that why I got the error.
